aiidalab/metadata.py

```diff
@@ -1,15 +1,17 @@
 """App metadata specification"""
 from __future__ import annotations
 
-from configparser import ConfigParser
+from configparser import ConfigParser, SectionProxy
```

Comment:
```suggestion
from typing import TYPE_CHECKING

if TYPE_CHECKING:
    from configparser import ConfigParser, SectionProxy
    from pathlib import Path
```

A minor nitpick, you are the expert on module loading performance, will this improve the loading efficiency?

Reply:
Not really, both of these are stdlib modules, which are generally very fast to import. More importantly, `pathlib` is getting imported all over the place anyway, and you would anyway need to import `ConfigParser` here.